docs/src/group.md

```diff
@@ -11,29 +11,11 @@ MPI.Group
 MPI.Comparison
```

Comment:
All docstrings I'm removing from this file were added in #521, but these functions/constants never had docstrings.